### PR TITLE
test(sdk): Testing the component library

### PR DESCRIPTION
**Pull request by Ark-kun**
Verifying that the component authoring library can work standalone.

### CODE REVIEW

File: test/presubmit-tests-sdk.sh [hunk 1]
```diff
@@ -21,6 +21,12 @@ python3 -m pip install mock  # TODO: Remove when added to requirements
 python3 -m pip install -r "$source_root/sdk/python/requirements.txt"
 python3 -m pip install --upgrade protobuf
 
+# Testing the component authoring library
+pushd "$(mktemp --directory)"
+cp "$source_root"/sdk/python/{components,components_tests} .
```
Comment (NikeNano) [inline test/presubmit-tests-sdk.sh:26]:
```suggestion
cp "$source_root"/sdk/python/kfp/{components,components_tests} .
```

I think this is the reason the tests fails as well

``` 
cp: cannot stat '/home/prow/go/src/github.com/kubeflow/pipelines/sdk/python/components': No such file or directory
cp: cannot stat '/home/prow/go/src/github.com/kubeflow/pipelines/sdk/python/components_tests': No such file or directory
```

Comment (Ark-kun) [reply test/presubmit-tests-sdk.sh]: Fixed. Thank you.

File: test/presubmit-tests-sdk.sh [hunk 2]
```diff
@@ -21,6 +21,12 @@ python3 -m pip install mock  # TODO: Remove when added to requirements
 python3 -m pip install -r "$source_root/sdk/python/requirements.txt"
 python3 -m pip install --upgrade protobuf
 
+# Testing the component authoring library
+pushd "$(mktemp --directory)"
+cp "$source_root"/sdk/python/kfp/{components,components_tests} .
```
Comment (NikeNano) [inline test/presubmit-tests-sdk.sh:26]:
```suggestion
cp -r "$source_root"/sdk/python/kfp/{components,components_tests} .
```

Comment (Ark-kun) [reply test/presubmit-tests-sdk.sh]: Fixed. Thanks.

**Pull request merged**